src/fixed-footer-layout.tsx

```diff
@@ -163,6 +163,7 @@ const FixedFooterLayout = ({
                 {renderBackground()}
                 {children}
             </div>
+            {isFooterVisible && <style>{`html {scroll-padding-bottom: ${domFooterHeight}px};`}</style>}
```

Comment:
I think this is a better condition:
```suggestion
            {isFooterVisible && isFooterFixed && <style>{`html {scroll-padding-bottom: ${domFooterHeight}px}`}</style>}
```
Also, the last `;` is not needed